wgpu-types/src/lib.rs

```diff
@@ -1911,26 +1914,24 @@ pub enum TextureFormat {
     Rgba32Float,
 
     // Depth and stencil formats
-    /// Special depth format with 32 bit floating point depth.
-    #[cfg_attr(feature = "serde", serde(rename = "depth32float"))]
-    Depth32Float,
-    /// Special depth/stencil format with 32 bit floating point depth and 8 bits integer stencil.
-    #[cfg_attr(feature = "serde", serde(rename = "depth32float-stencil8"))]
-    Depth32FloatStencil8,
+    /// Stencil format with 8 bit integer stencil.
+    #[cfg_attr(feature = "serde", serde(rename = "stencil8"))]
+    Stencil8,
```

Comment:
I'm a bit apprehensive to add stencil8 as I seem to remember that we need to fall back to other formats (like Depth24PlusStencil8) in some cases and because stencil8 isn't behind a feature we actually need to handle all the cases before we can expose it at all.

Reply:
Oki - let's remove it for now - we'll add it in a following submit to do also deep testing on it